docs/about.rst

```diff
@@ -0,0 +1,31 @@
+About
+=====
+
+Disclaimer
+----------
+
+Naggum don't aim to be yet another Common Lisp or Scheme or whatever
+implementation. Instead, we are trying to deliver a modern Lisp dialect that
+makes use of most of CLI benefits.
+
+Features
+--------
+
+- low level and high level features coupled together
```

Comment:
1. I'm not sure how to find any evidence to confirm that, but it appears to be that adjectives consisting of multiple words are usually hyphenated, i.e. "low-level technology", but "this technology works on the low level".

2. "Coupled together" sounds a bit like tautology to me. If they're coupled, they're already together, aren't they?

3. I don't understand the meaning of low- and high-level technologies being coupled. You mean that the language is equally suitable both for bit-twiddling and stuff like implementing map-reduce?